src/Mod/Sketcher/Gui/SketcherTransformationExpressionHelper.cpp

```diff
@@ -0,0 +1,143 @@
+/***************************************************************************
+ *   Copyright (c) 2025 FreeCAD Contributors                               *

```

Comment:
That's probably not actually a legally-valid copyright claim. Either it's copyright you, or it's copyright someone you assign the copyright to (I personally assign most of my work to "The FreeCAD project association AISBL"). Also, this is the old copyright block, with the physical mailing address for the FSF -- they don't want to use that anymore.